### ISSUE TITLE
where download pretrained model?

### ISSUE THREAD

**Issue by BDMLLang** [hq-jiang/instance-segmentation-with-discriminative-loss-tensorflow#11]
where download weights files?  thanks!
